Adds wycats' Octane tutorial Part 3 & 4 (#627)
ember-learn/ember-blog: source/2020-05-01-the-ember-times-issue-146.md

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹

<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>

> Yehuda's Octane Tutorial continued 🔥,

Reviewer: ```suggestion
Yehuda's Octane tutorial continued 🔥,
```

  
READMORE

---

## [Yehuda's Octane tutorial continued 🔥](https://yehudakatz.com/2020/03/25/ember-octane-lets-go/)

> Our very own [Yehuda Katz (@wycats)](https://github.com/wycats) wrote a blog series on [Octane tutorial](https://yehudakatz.com/2020/03/25/ember-octane-lets-go/), and there are two new posts added to it.

Reviewer 2: What do you think about the following wording?

```suggestion
Our very own [Yehuda Katz (@wycats)](https://github.com/wycats) wrote a blog series for an [Octane tutorial](https://yehudakatz.com/2020/03/25/ember-octane-lets-go/), and there are two new posts added to it.
```

Author: Yeah sounds good now @jayjayjpg .

> The third part of the tutorial, [Pulling Out Data](https://yehudakatz.com/2020/03/30/ember-octane-a-data-file/), talks about moving data from HTML into a separate file. This is the part in the series that introduces you to writing Javascript for the first time. You'll also learn how to build your helpers and writing unit tests for them.

Reviewer 2: I think it'd be

```suggestion
The third part of the tutorial, [Pulling Out Data](https://yehudakatz.com/2020/03/30/ember-octane-a-data-file/), talks about moving data from HTML into a separate file. This is the part in the series that introduces you to writing Javascript for the first time. You'll also learn how to build your helpers and write unit tests for them.
```

What do you think?

Author: Yeah! What you have suggested sounds perfect!
